app/controllers/fast_controller.rb

```diff
@@ -78,49 +63,25 @@ def num_records
     Settings.autocomplete_lookup.num_records
   end
 
-  # Split into left-anchored matches to query and other matches.
-  def partition_suggestions(suggestions, query)
-    Array(suggestions).partition { |suggestion| suggestion.keys.first.downcase.start_with?(query.downcase) }
-  end
-
-  def merge_suggestions(suggestions1, suggestions2, query)
-    left_match_suggestions1, other_suggestions1 = partition_suggestions(suggestions1, query)
-    left_match_suggestions2, other_suggestions2 = partition_suggestions(suggestions2, query)
-
-    # Order is left match (non-wildcard) alpha, left match (wildcard) alpha, other matches alpha
-    left_match_suggestions = (sort_suggestions(left_match_suggestions1) + sort_suggestions(left_match_suggestions2))
-                             .take(num_records)
-    num_other_suggestions = num_records - left_match_suggestions.size
-    other_suggestions = (other_suggestions1 + other_suggestions2)
-                        .uniq
-                        .take(num_other_suggestions)
-
-    left_match_suggestions + sort_suggestions(other_suggestions)
-  end
-
-  def sort_suggestions(suggestions)
-    suggestions.sort { |suggestion1, suggestion2| suggestion1.keys.first <=> suggestion2.keys.first }
-  end
-
   URI_PREFIX = 'http://id.worldcat.org/fast/'
 
   def key(idroot, type)
     "#{URI_PREFIX}#{idroot.delete_prefix('fst').to_i}/::#{cocina_type(type)}"
   end
 
   # Map of FAST types to Cocina types.
-  SUBJECT_TYPES = {
-    'person' => 'person',
-    'corporate' => 'organization',
-    'event' => 'event',
-    'geographic' => 'place',
-    'title' => 'title',
-    'form' => 'genre',
-    'meeting' => 'conference',
-    'chronological' => 'time'
+  TAG_TYPES = {
+    100 => 'person',
+    110 => 'organization',
+    111 => 'event',
+    151 => 'place',
+    130 => 'title',
+    155 => 'genre',
+    148 => 'time'
+    # NOTE: Missing: 'meeting' => 'conference'
```

Comment:
@arcadiafalcone AFAICT, the new FAST API does not distinguish between event-type authorities and meeting-type authorities. Is this problematic? Did I get it wrong?

Reply:
Going from the corresponding MARC tags, 111 should be meeting/conference and 147 should be event. The data set downloads and the web search both still make the distinction between the two types, so perhaps the API documentation is in error?

Some sample values:
Paris Peace Conference (1919-1920) http://id.worldcat.org/fast/1696461 is a meeting
World War (1914-1918) http://id.worldcat.org/fast/1180746 is an event

Reply:
Ah, awesome, thank you. 😄